src/io/bt.js

```diff
@@ -115,7 +117,10 @@ class BT extends JSONRPCWebSocket {
     _sendError (/* e */) {
         this.disconnect();
         // log.error(`BT error: ${JSON.stringify(e)}`);
-        this._runtime.emit(this._runtime.constructor.PERIPHERAL_ERROR);
+        this._runtime.emit(this._runtime.constructor.PERIPHERAL_ERROR, {
+            message: `Scratch lost connection to`,
+            extensionId: this._extensionId
```

Comment:
I think we might want to define this message in the GUI and only pass `extensionId` as the data of this. We might want the message to be different in different contexts.

Reply:
Yes, this suggestion feels better to me, I wasn't sure what that might look like on the GUI side.